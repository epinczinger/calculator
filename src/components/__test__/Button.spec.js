import React from 'react';
import Button from '../Button';
import renderer from 'react-test-renderer';

describe('The Button', () => {
    it('renders as expected matching snapshot', () => {
        const tree = renderer.create(<Button name='AC' clickHandler={Function} />).toJSON();
        expect(tree).toMatchSnapshot();
    } )
});