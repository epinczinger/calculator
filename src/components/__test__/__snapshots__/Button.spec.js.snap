// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Button renders as expected matching snapshot 1`] = `
<button
  className="w-full h-full"
  onClick={[Function]}
  type="button"
>
  AC
</button>
`;
